MainApp/PassengerRegistration/Security.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace PassengerRegistration
+{
+    class Security
+    {
+        public static bool SecurityQuestions()
+        {
+            string resultForbiddenItems;
+            string resultPermission;
+
+            Console.WriteLine(Constants.SecurityQuestions.ForbiddenItems);
+            resultForbiddenItems = Console.ReadLine().ToUpper();
+
+            if (resultForbiddenItems == "Y")
```

Comment:
do not use `==` operator with strings comparison. Just use string.Equals() method